lib/constants/orgScopes.js

```diff
@@ -1,4 +1,11 @@
 import { ALL } from 'lib/constants/scopes';
+import {
```

Comment:
Lodash not used in this file?

Reply:
Looks like this needs rebasing from develop for the build to pass. Also probably failing linting. Otherwise looks good :)